integrations/Drip/browser.js

```diff
@@ -0,0 +1,161 @@
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import { removeUndefinedAndNullValues } from "../utils/commonUtils";
+
+class Drip {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.campaignId = config.campaignId;
+    this.name = "DRIP";
+  }
+
+  init() {
+    logger.debug("===In init Drip===");
+
+    window._dcq = _dcq || [];
+    window._dcs = _dcs || {};
+    window._dcs.account = this.accountId;
+
+    (function () {
+      var dc = document.createElement("script");
+      dc.type = "text/javascript";
+      dc.async = true;
+      dc.src = `//tag.getdrip.com/${this.accountId}.js`;
+      var s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(dc, s);
+    })();
+  }
+
+  identify(rudderElement) {
+    logger.debug("===In Drip identify===");
+
+    const { message } = rudderElement;
+    if (!message.context.traits) {
+      logger.error("user traits not present");
+      return;
+    }
+
+    const { email } = message.context.traits;
+    if (!email) {
+      logger.error("email is required for the call");
+      return;
+    }
+
+    const { euConsent } = message.context.traits;
+    if (
+      euConsent &&
+      !(
+        euConsent.toLowercase() === "granted" ||
+        euConsent.toLowercase() === "denied"
+      )
+    ) {
+      euConsent = "";
```

Comment:
if we set empty string, it wont be removed actually by `removeUndefinedAndNullValues` function